### PR TITLE
Modified various Chinese translations

### PR DESCRIPTION
**Pull request by Liuxi-G**
Hi there,I have found your project very interesting, however the translation in chinese have many faults so I modified some of the translation,and added some setences that haven't translated.
I hope you like this version of translation and accept my PR :)
Thank you!

### CODE REVIEW

File: app/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -10,91 +10,116 @@
...
 		<a href="https://tldrlegal.com/license/mit-license">MIT 许可证</a>。
 	</string>
-	<string name="credits_content5">谢谢<a href="https://github.com/psyanite">Yan Tsui</a>
-		为了测试和设计&appname;图标.
+	<string name="credits_content5">感谢<a href="https://github.com/psyanite">Yan Tsui</a>
+		对测试和设计的贡献&appname;icon.
 	</string>
-	<string name="credits_content6"> <a href="https://github.com/wardbradt">Ward Bradt</a>
-		把&appname;从英文翻译成中文。
+	<string name="credits_content6"> 此版本翻译&appname;由<a href="https://github.com/wardbradt">Liuxi-G</a>
```
Comment (ScreamingHawk) [inline app/src/main/res/values-zh/strings.xml]: Could you please keep the original translator here and add your own `<a href="your_website_link">Luixi-G</a>` as well. I'd like to keep both translators, even if you are changing most of it :)

Comment (Liuxi-G) [reply app/src/main/res/values-zh/strings.xml]: Hate to say this but as a native chinese speaker the original translation seems to be google translated,sure if you want to keep the original one,I can send a new pull request.

**Comment by Liuxi-G** [comment]
I have kkeped the original translator as you suggested :)

**Comment by Liuxi-G** [comment]
I have keeped the oroginal translator as you asked :)

File: app/src/main/res/values-zh/strings.xml [hunk 2]
```diff
@@ -53,7 +53,10 @@
 	<string name="credits_content5">感谢<a href="https://github.com/psyanite">Yan Tsui</a>
 		对测试和设计的贡献&appname;icon.
 	</string>
-	<string name="credits_content6"> 此版本翻译&appname;由<a href="https://github.com/wardbradt">Liuxi-G</a>
+	
+        <string name="credits_content6"> <a href="https://github.com/wardbradt">Ward Bradt</a>
+		把&appname;从英文翻译成中文。
+        <string name="credits_content6"> &appname的此版本翻译;由<a href="https://github.com/Liuxi-G">Liuxi-G</a>
```
Comment (ScreamingHawk) [inline app/src/main/res/values-zh/strings.xml]: Missing a `;` after `&appname`. Not sure if the `;` further on is intentional or the misplaced one so I'll have to get you to fix it again. Sorry :|

**Comment by ScreamingHawk** [comment]
I saw your comment before that the original translation appeared Google translated. While that's entirely possible I have no way to verify it as I don't speak the language... 

There have been a couple cases where people have claimed the same thing but I can't really take sides without more information so prefer to give the benefit of the doubt and credit everyone who tried. 

That being said you could be berating them in the translated credit notes and I'd have no idea :P

**Comment by Liuxi-G** [comment]
Already modified the mistake. thank you for finding that small  error,Icompletely understand your concern,Hope this one is perfect now,

**Comment by ScreamingHawk** [comment]
Perfect! Thank you very much. 

Like I said before, I can never be sure the translation is correct but with more eyes comes more certainty. I really appreciate you taking the time to submit this pull request. It might seem trivial to you, but there's absolutely no way it would happen without your help.

**Comment by ScreamingHawk** [comment]
Issue reference: #26

**Pull request merged**